src/CMakeLists.txt

```diff
@@ -129,22 +130,43 @@ if(Cython_FOUND)
     foreach(file ${CYCLUS_CORE_CYTHON_FILES})
         message(STATUS "Cython Compiling ${file}")
         get_filename_component(name ${file} NAME_WE)
-        set(_generated_h "${CMAKE_CURRENT_SOURCE_DIR}/${name}.cc.h")
-        set(_generated_cc "${CMAKE_CURRENT_SOURCE_DIR}/${name}.cc.gen")
+        set(_generated_h "${CMAKE_CURRENT_SOURCE_DIR}/${name}.cpp.h")
+        set(_generated_cpp "${CMAKE_CURRENT_SOURCE_DIR}/${name}.cpp.gen")
```

Comment:
Are these changes substantive or just aesthetic?

Reply:
Substantive.  CMake grabs all of the  `.cc` files in `src/` and compiles them into `libcyclus.so` but I don't want it to include the cython modules since those are built as their own respective shared libraries.  Renaming them to `*.cpp` is one way to do this, though I can just modify the GLOB that grabs the files to compile if we want to keep consistent file extensions.

Reply:
Where is that GLOB?  I feel that modifying the GLOB may be more explicit and leave a better record of its importance.  It may be difficult for future me (or others) to recognize the importance of this file extension difference.